face_detection_masking/requirements.txt

```diff
@@ -1,7 +1,53 @@
+attrs==20.3.0
+cachetools==4.2.1
+certifi==2020.12.5
+cffi==1.14.5
+chardet==4.0.0
+click==7.1.2
+cloudevents==1.2.0
+coverage==5.5
+darkflow==1.0.0
+deprecation==2.1.0
 Flask==1.0.2
 functions-framework==2.1.0
-google-cloud-storage
+google-api-core==1.26.2
+google-auth==1.28.0
+google-cloud-core==1.6.0
+google-cloud-storage==1.37.0
 google-cloud-vision==2.2.0
-opencv-python==4.3.0.38
+google-crc32c==1.1.2
+google-resumable-media==1.2.0
+googleapis-common-protos==1.53.0
+grpcio==1.36.1
+gunicorn==20.1.0
+idna==2.10
+iniconfig==1.1.1
+itsdangerous==1.1.0
+Jinja2==2.11.3
+libcst==0.3.17
+MarkupSafe==1.1.1
+mypy-extensions==0.4.3
 numpy==1.19.1
-pytest==6.2.2
\ No newline at end of file
+opencv-python==4.3.0.38
+packaging==20.9
+pluggy==0.13.1
+proto-plus==1.18.1
+protobuf==3.15.6
+py==1.10.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycparser==2.20
+pyparsing==2.4.7
+pytest==6.2.2
+pytest-cov==2.11.1
+pytz==2021.1
+PyYAML==5.4.1
+requests==2.25.1
+rsa==4.7.2
+six==1.15.0
+toml==0.10.2
+typing-extensions==3.7.4.3
+typing-inspect==0.6.0
+urllib3==1.26.4
+watchdog==1.0.2
+Werkzeug==1.0.1
```

Comment:
this seems like a whole lot of unnecessary modules

Reply:
I am just pip freeze to requirements

Reply:
yea your pip freeze must have included modules from your pip environment since most of these are not used. Let's revert this :D